Source_Code/TM_Backend/TeamMentor.Users/Users/Users_Creation.cs

```diff
@@ -239,6 +249,38 @@ public static Signup_Result createTmUserResponse(this TM_UserData userData, NewU
         }
 
         #region Sigup validations, shorter and specialized methods
+        private static  bool IsValidEmailAddress(string emailAddress)
+        {
+            bool isValid = false;
+            try
+            {
+                var address = new System.Net.Mail.MailAddress(emailAddress);
+                isValid = true;
+            }
+            catch
+            {
+                isValid = false;
+            }
+            return isValid;
+        }
+        private static Signup_Result ValidateEmailPattern(TMConfig config)

```

Comment:
the name of this function is not right, since there is no validation happening here, right?

Doesn't this function returns the error message to be send to the user?